### PR TITLE
SSCSSI-348 Add correspondence topic to existing namespace

### PR DESCRIPTION
**Pull request by adusumillipraveen**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/SSCSSI-348

### Change description ###

This doesn't impact any existing functionality in prod.

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by sanjayparekh-hmcts — APPROVED** [review]

File: servicebus.tf [hunk 1]
```diff
@@ -99,3 +101,35 @@ resource "azurerm_key_vault_secret" "evidence_share_topic_primary_shared_access_
...
 }
+
+module "correspondence-topic" {
+  source                                  = "git@github.com:hmcts/terraform-module-servicebus-topic?ref=master"
+  name                                    = local.correspondence_topic_name
+  namespace_name                          = local.servicebus_namespace_name
+  resource_group_name                     = local.resource_group_name
+  requires_duplicate_detection            = true
+  duplicate_detection_history_time_window = "PT1H"
+  max_size_in_megabytes                   = 2048
```
Comment (nitinprabhuhmcts) [inline servicebus.tf:112]: do you really need 2MB ?

Comment (adusumillipraveen) [reply servicebus.tf:112]: couldn't think of ideal size, Should i say 1 MB ? its just max anyway, we are not charged for that setting.

Comment (nitinprabhuhmcts) [reply servicebus.tf:112]: cool should be good then

**Review by nitinprabhuhmcts — APPROVED** [review]

**Review by SomeshRA — COMMENTED** [review]
LGTM

**Pull request merged**